docs/src/main/tut/docs/core/algebras/README.md

```diff
@@ -111,10 +145,41 @@ def myService[F[_]](implicit userRepository: UserRepository[F]) = ???
 def myService2[F[_]: UserRepository] = ???
 ```
 
+## Composed Operations 
+
+The trait `EffectLike[F[_]]` mentioned above, apart from the type alias `FS[_]`, also defines two type aliases `FS.Seq[_]` and `FS.Par[]`. 
+
+```Scala
+trait EffectLike[F[_]] {
+  final type FS[A] = FreeS.Par[F, A]
+  final object FS {
+    final type Seq[A] = FreeS[F, A]
+    final type Par[A] = FreeS.Par[F, A]
+  }
+}
+```
+
+These type aliases can be used to define some `FS` operations that are derived from other operations. , by combining other operations: 
+
+* `FS.Par[A]` is an alias for `FreeApplicative[F, A]`. You can declare a derived operation of type `FS.Par` by applying the methods of the `Functor` and `Applicative` type classes. 
+* `FS.Seq[A]` is an alias for `Free[FreeApplicative[F, ?], A]`, that can be combined using the methods of the `Monad` type class.
+
+```tut:book
+import cats.syntax.monad._
+@free trait X { 
+  def a: FS[Int]
+  def b(i: Int): FS.Par[Int] = a.map(x => x+i) 
```

Comment:
format `x+i` as `x + i` etc to be consistent with the code base and other examples. Sadly we can't run scalafmt in tut code blocks.